navigation-material/src/androidTest/java/com/google/accompanist/navigation.material/BottomSheetNavigatorTest.kt

```diff
@@ -273,9 +296,6 @@ internal class BottomSheetNavigatorTest {
 
         assertThat(navigatorState.transitionsInProgress.value)
             .doesNotContain(backStackEntry1)
-
-        assertThat(navigatorState.transitionsInProgress.value)
```

Comment:
Is there a reason this was removed? I would think we still want to ensure after `awaitIdle` completes that `backStackEntry2` has been removed from the transitioning set and moved all the way to `RESUMED`.

Reply:
Ack, not sure why I removed it.